docs/mp/oci/02_OciVault.adoc

```diff
@@ -0,0 +1,233 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2021 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+:javadoc-base-url-api: {javadoc-base-url}io.helidon.config/io/helidon/oci
+
+= The OCI Vault
+:h1Prefix: MP
+:description: Helidon OCI Vault integration
+:keywords: oci, vault
+:common-deps-page-prefix-inc: ../../shared/dependencies/common_shared.adoc
+:feature-name: OCI Vault
+
+The Helidon MP OCI Vault integration provides easy access to Oracle Cloud Vault features.
+
+include::{common-deps-page-prefix-inc}[tag=maven-dependency]
+
+[source,xml]
+----
+        <dependency>
+            <groupId>io.helidon.integrations.oci</groupId>
+            <artifactId>helidon-integrations-oci-cdi</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.helidon.integrations.oci</groupId>
+            <artifactId>helidon-integrations-oci-vault</artifactId>
+        </dependency>
+----
+
+
+== Setting up the OCI Vault
+
+
+In order to use the OCI Vault integration, the following setup should be made.
+
+* The configuration required for Vault integration includes:
+* Vault OCID - to use the correct Vault, as more than one can be configured
+* Compartment OCID - OCI specific compartment
+* Encryption Key OCIO - required when doing encryption/decryption
+* Signature Key OCID - required when doing signatures/verification
+* Cryptographic endpoint - required for all except secrets
+
+First specify OCID is of Vault items:
+
+[source,yaml]
+----
+oci:
+  vault:
+    vault-ocid: "<...>"
+    compartment-ocid: "<...>"
+    encryption-key-ocid: "<...>"
+    signature-key-ocid: "<...>"
+    cryptographic-endpoint: "<...>"
+----
+
+The OCIDs can be set up and found in OCI under Security tab.
```

Comment:
Again it would be good to show a picture from OCI console.

Reply:
I've captured the screens.

Reply:
![vault_ocids](https://user-images.githubusercontent.com/31409174/117030506-b9c68380-accd-11eb-9863-186daefbae25.png)

Reply:
done